src/gen_smtp_client.erl

```diff
@@ -47,12 +47,14 @@
 -include_lib("eunit/include/eunit.hrl").
 -compile(export_all).
 -else.
--export([send/2, send/3, send_blocking/2]).
+-export([send/2, send/3, send_blocking/2, open/1, deliver/2]).
```

Comment:
No `close/1` function?
How do we clean up an opened connection?
(I assume I can use `deliver/1` multiple times on an open connection)